example_test.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright The ORAS Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package credentials_test
+
+import (
+	"context"
+
+	credentials "github.com/oras-project/oras-credentials-go"
+	"oras.land/oras-go/v2/registry/remote/auth"
+)
+
+func ExampleNewNativeStore() {
+	// For the argument of NewNativeStore, use "wincred" for Windows,
+	// "pass" for linux and "osxkeychain" for macOS
+	ns := credentials.NewNativeStore("pass")
+	err := ns.Put(context.Background(), "localhost:8080",
+		auth.Credential{
+			Username: "username-example",
+			Password: "password-example"})
+	if err != nil {
+		panic(err)
+	}
+}
+
+func ExampleNewFileStore() {
+	fs, err := credentials.NewFileStore("example/path/config.json")
+	if err != nil {
+		panic(err)
+	}
+	err = fs.Put(context.Background(), "localhost:8080",
+		auth.Credential{
+			Username: "username-example",
+			Password: "password-example"})
+	if err != nil {
+		panic(err)
+	}
+}
+
+func ExampleNewStore() {
+	s, err := credentials.NewStore("example/path/config.json", credentials.StoreOptions{AllowPlaintextPut: true})
```

Comment:
Can we have a comment to explain the default behavior of `NewStore` and when to use `credentials.StoreOptions{AllowPlaintextPut: true}` ? 
Reference: https://pkg.go.dev/github.com/oras-project/oras-credentials-go#StoreOptions

Reply:
Added summarized comments for this part.